database/indexC.js

```diff
@@ -0,0 +1,53 @@
+const cassandra = require('cassandra-driver');
+const Promise = require('bluebird');
+const fs = require('fs');
+
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1:9042'],
+  keyspace: 'robinhood',
+  promiseFactory: Promise.fromCallback,
+});
+
+console.time('Seed C');
+console.log('SEEDING CASSANDRA START');
+const runSeed = (i = 0) => {
+  if (i < 26) {
```

Comment:
you could add some line spaces for readability to break up the block. not major, just a preference